### ISSUE TITLE
Output issues since 5.0.2

### ISSUE THREAD

**Issue by kaleidografik** [postcss/postcss-nested#112]
Hi guys. I've been using your plugin for a while now (combined with tailwindcss), but have just noticed an issue when updating beyond v5.0.1

The below code no longer correctly outputs - can't see anything in the release notes that would cause an issue but wondering if you might notice anything that might be causing issue?

Thanks

```
input,
  textarea {
    @apply appearance-none outline-none bg-transparent;

    // States
    &:focus {
      @apply shadow-none outline-none;
    }

    // Placeholder
    &::placeholder {
      @apply text-transparent; // Invisible placeholders, just used to detect if content entered in field
    }

    // Input is empty
    &:placeholder-shown {
      @apply text-black;
    }

    // Input is not empty
    &:not(:placeholder-shown) {
      @apply text-blue-slate;

      // Floating labels (on certain input types)
      &[type=text],
      &[type=email],
      &[type=tel],
      &[type=password] {

        ~ label {
          @apply pt-0 text-sm;
        }
      }
    }
}
```

**Comment by ai** [comment]
PR is welcome. I burned out on the PostCSS project.

**Comment by ai** [comment]
Should be fixed with PostCSS 8.3.3.
